New package: wluma-4.9.0

realcharmer opened the pull request:
Automatic brightness adjustment based on screen contents and ALS.
https://github.com/maximbaz/wluma

#32485, #31431

#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

realcharmer commented:
Updated to `4.4.0`.

Calandracas606 commented on srcpkgs/wluma/template:
```diff
@@ -0,0 +1,19 @@
+build_style=cargo
+hostmakedepends="pkg-config clang"
+makedepends="eudev-libudev-devel v4l-utils-devel wayland-devel Vulkan-Tools
+ Vulkan-Headers dbus-devel"
```
```suggestion
 Vulkan-Headers dbus-devel vulkan-loader-devel"
```

Calandracas606 commented:
package should install the udev rules, manpages, etc.

realcharmer commented:
Sorry for having leaving this unattended for a while. I have:

- Fixed the build issue with missing dependency
- Updated to v4.6.1
- Added UDEV rules

Regarding manapages, they are built using `marked-man`, which is not available at the moment.

realcharmer commented:
Updated to v4.9.0.

Duncaen commented on srcpkgs/wluma/template:
```diff
@@ -0,0 +1,23 @@
+}
+
+post_install() {
+	vlicense LICENSE
```
You have duplicate `post_install` functions, the first one will be skipped afaik.

realcharmer replied:
How did i miss this?! 😅
